Statement that cache can be enabled|disabled only from command line starting Magento 2.2 is false #4846 (#4878)
magento/devdocs: guides/v2.2/config-guide/cli/config-cli-subcommands-cache.md

This command enables you to enable or disable all cache types or only the ones you specify. Disabling cache types is useful during development because you see the results of your changes without having to flush the cache; however, disabling cache types has an adverse effect on performance.

{:.bs-callout .bs-callout-info}

> Starting in version 2.2, you can enable or disable cache types only using the command line while running Magento in production mode. Also when Magento runing in developer mode, then you can enable or disable cache types using command line or manually. Before doing so, you must manually make `<magento_root>/app/etc/env.php` writeable by the [Magento file system owner]({{ page.baseurl }}/install-gde/prereq/file-system-perms.html). You can clean (also referred to as _flush_ or _refresh_) cache types using either the command line or the [Magento Admin](https://glossary.magento.com/magento-admin).

Reviewer: ```suggestion
Starting in version 2.2, you can only enable or disable cache types using the command line while running Magento in production mode. If running Magento in developer mode, you can enable or disable cache types using the command line or manually. Before doing so, you must manually make `<magento_root>/app/etc/env.php` writeable by the [Magento file system owner]({{ page.baseurl }}/install-gde/prereq/file-system-perms.html). You can clean (also referred to as _flush_ or _refresh_) cache types using either the command line or the [Magento Admin](https://glossary.magento.com/magento-admin).
```
